features/ultils/world.ts

```diff
@@ -36,6 +42,54 @@ export class OurWorld extends World {
       destFilePath: filePath,
     });
   }
+
+  public getCredentialByAppKey(appKey: string): Credential {
+    const credential = this.credentials?.find((c) => c.key === appKey);
+    if (credential === undefined) {
+      throw new Error(`The credential with app key ${appKey} is not found`);
+    }
+
+    if (!credential.appId) {
+      throw new Error(`The credential with app key ${appKey} has no App ID`);
+    }
+
+    return credential;
+  }
+
+  public getAPITokenByAppAndPermission(
+    appKey: string,
+    permission: {
```

Comment:
It's better to be `permissions`

```suggestion
  public getAPITokenByAppAndPermissions(
    appKey: string,
    permissions: {
```